src/Common/src/Interop/Windows/Kernel32/Interop.QueryDosDevice.cs

```diff
@@ -11,7 +11,10 @@ internal partial class Interop
 {
     internal partial class Kernel32
     {
-          [DllImport(Libraries.Kernel32, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
-          internal static extern uint QueryDosDeviceW(string lpDeviceName, IntPtr lpTargetPath, int ucchMax);
+        [DllImport(Libraries.Kernel32, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Unicode)]
+        internal static extern uint QueryDosDeviceW(string lpDeviceName, IntPtr lpTargetPath, int ucchMax);
```

Comment:
is this overload used anywhere? I'd rather expose only `char*` overload than having two

Reply:
I would have thought it was used quite a few times, so I left it there for reference,
But to my surprise QueryDosDevice() isn't use that often.

Ive remove the one not in use.